internal/bminventory/inventory.go

```diff
@@ -1647,10 +1647,7 @@ func (b *bareMetalInventory) RegisterHost(ctx context.Context, params installer.
 			b.eventsHandler.AddEvent(ctx, params.ClusterID, params.NewHostParams.HostID, models.EventSeverityError,
 				"Failed to register host: cluster cannot accept new hosts in its current state", time.Now())
 			return installer.NewRegisterHostForbidden().
```

Comment:
Maybe i wasn't clear
```suggestion
			return common.NewApiError(http.StatusForbidden, err)
```